src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs

```diff
@@ -80,36 +84,24 @@ IConfiguration configuration
             int prNumber,
             string hostName,
             string codeFileName = null,
-            string baselineCodeFileName = null)
+            string baselineCodeFileName = null,
+            bool commentOnPR = true)

```

Comment:
Where is the case where you pass false here? Or will that come later.

Reply:
If caller of the script wants to suppress commenting on PR by APIView then this flag can be passed as false. Regular SDK pipeline will continue with default true so APIView will comment on the PR as it does now. But swagger pipeline can pass flag as false so comment can be created by swagger pipeline

Reply:
Done. I have pushed the change in controller.

Reply:
Caller of the REST API can decide whether we need APIView to comment on PR or not. Swagger pipeline(SDK automation) will be the consumer of this change.